### PR TITLE
Misc queue from 2018 (sorry, lost sources)

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6204,14 +6210,18 @@ feeded->fed
 feetur->feature
 feeture->feature
 feld->field
+felisatus->felicitous
 femminist->feminist
 fempto->femto
+feonsay->fianc&eacute;e
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: remove

or could maybe be unicode if we use unicode elsewhere

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Thanks. Fixed.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7553,14 +7559,18 @@ feeded->fed
...
 fempto->femto
+feonsay->fiancée
 fequency->frequency
 feromone->pheromone
+fertil->fertile
 fertily->fertility
 fetaure->feature
 fetaures->features
 feture->feature
+fewd->feud
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add few

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -7544,6 +7549,7 @@ featurs->features
 feautre->feature
 feauture->feature
 feautures->features
+febewary->february
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Capitalise the month name correction.

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -7525,15 +7525,20 @@ farmework->framework
...
 fatser->faster
 fature->feature
 faught->fought
 fauilures->failures
 faund->found
+fave->save
 favoutrable->favourable
+faymus->famous
 fcound->found
+feasabile->feasible
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Could add feasability->feasibility

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**